cli/ops/os.rs

```diff
@@ -101,6 +101,22 @@ pub fn op_env(
   Ok(JsonOp::Sync(json!(v)))
 }
 
+#[derive(Deserialize)]
+struct GetEnv {
+  key: String,
+}
+
+pub fn op_get_env(
+  state: &ThreadSafeState,
+  args: Value,
+  _zero_copy: Option<PinnedBuf>,
+) -> Result<JsonOp, ErrBox> {
+  let args: GetEnv = serde_json::from_value(args)?;
+  state.check_env()?;
+  let v = env::var(args.key)?;
```

Comment:
error is caused by this `?`

Reply:
No it's because we don't have an ErrorKind mapping defined for [VarError](https://doc.rust-lang.org/std/env/enum.VarError.html).
I also think that if the env var is not found, we should simply return undefined.
That seems in line with node, python etc.

Reply:
For that matter, this API would be better if it was like the `Map` object. So getting a single env var would be done with `Deno.env.get("bla")` and iteration would be `Deno.env.entries()`. 
But 🤷‍♂ I don't have time for that now.